### PR TITLE
Lesson 24

### PR DESCRIPTION
**Pull request by daianequeiroz**

### CODE REVIEW

File: Exercises/JS/07-js-loops/js/script.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+for (let i = 1; i < 1000; i++) {
+   let result = +i;
```
Comment (chriztie) [inline Exercises/JS/07-js-loops/js/script.js]: The operators should be swapped here. When you do this, the value of `i` will be assigned to result.
Objective of this exercise is to break the loop when the intermediate result exceeds 400.
If you want the value of `i` to be added to `result`, then the variable should be declared elsewhere.
Keep in mind,
`result += i` is the shortened form of `result = result + i` and for this, result should be declared somewhere else. Hope I'm clear if not please let me know.

**Review by chriztie — CHANGES_REQUESTED** [review]

File: Exercises/JS/07-js-loops/js/script.js (hunk 1 above)
Comment (daianequeiroz) [reply Exercises/JS/07-js-loops/js/script.js]: I understood and fixed it.

**Review by chriztie — APPROVED** [review]
Yep good job @daianequeiroz 💯

File: Exercises/JS/07-js-loops/js/script.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+let result=0;
+for (let i = 1; i <= 1000; i++) {
+    if (result >= 400) break;
+    result++;
```
Comment (chriztie) [inline Exercises/JS/07-js-loops/js/script.js:6]: Nice

File: Exercises/JS/07-js-loops/js/script.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+let j = 100;
+while (j >= 0) {
+    console.log(j);
+    j--;
+ }
+
+//Show even numbers between 0 and 100
+for (let i = 2; i < 100; i++) {
+    if (i % 2 == 0) console.log(i);
```
Comment (chriztie) [inline Exercises/JS/07-js-loops/js/script.js:46]: Use brackets as a good practice when you have one statement 🙂 
Good job @daianequeiroz!

**Pull request merged**